stock_prices/stock_prices.py

```diff
@@ -3,7 +3,15 @@
 import argparse
 
 def find_max_profit(prices):
-  pass
+  max_p = None
+  for i in range(len(prices) - 1):
+    for j in range(i + 1, len(prices)):
+      prof = prices[j] - prices[i]
+      if not max_p:
+        max_p = prof
```

Comment:
same as the problem above, this is a very readable and clean solution.  Perhaps instead of having this extra if/else to check if max_p has been initialized yet, we could initialize it to something like prices[1]-prices[0] up on line 6.

Great work with these 2 problems and with working on your whiteboarding problem.  I would advise coming back to work on some of the other problems in this repo when you get a chance.